scripts/run_frontend_tests.py

```diff
@@ -64,25 +64,26 @@ def main(args=None):
         'on your filesystem.',
         'Running test in development environment'])
 
-    build.main(args=[])
 
-    cmd = [
-        os.path.join(common.NODE_MODULES_PATH, 'karma', 'bin', 'karma'),
-        'start', os.path.join('core', 'tests', 'karma.conf.ts')]
-
-    task = subprocess.Popen(cmd)
-    task.communicate()
-    task.wait()
-
-    if parsed_args.run_minified_tests is True:
+    if parsed_args.run_minified_tests:
```

Comment:
Wow -- I'm surprised we've had the old code for a while, with its two builds. Thanks for the refactor.